clang/lib/APINotes/APINotesYAMLCompiler.cpp

```diff
@@ -456,6 +458,7 @@ template <> struct MappingTraits<Tag> {
     IO.mapOptional("EnumKind", T.EnumConvenienceKind);
     IO.mapOptional("SwiftCopyable", T.SwiftCopyable);
     IO.mapOptional("Methods", T.Methods);
+    IO.mapOptional("Tags", T.Tags);
```

Comment:
Should this perhaps be ParentTags or something else? It's not particularly clear why you would list tags in a tag. I think that namespace would also be confusing.

Reply:
Hmm, but is it the full set of tags or the tags that have associated attributes? I think that I got confused due to the namespace being built the obvious front to end.

Reply:
I see now how I was confused. These are tags at a single level and are not recursively enumerated. I was thinking of it as the namespace path. I think that a comment might be sufficient.

Reply:
Yeap, I added a comment to clarify that.